lib/barcelona/plugins/datadog_plugin.rb

```diff
@@ -53,13 +66,41 @@ def add_files!(user_data)
             container_collect_all: true
           process_config:
             enabled: 'true'
+          runtime_security_config:
+            enabled: true
```

Comment:
I might miss something, but don't we need to add `remote_configuration:` key?
https://docs.datadoghq.com/security/cloud_security_management/setup/csm_cloud_workload_security/agent/linux/#:~:text=remote_configuration%3A%0A%20%20%23%23%20%40param%20enabled%20%2D%20boolean%20%2D%20optional%20%2D%20default%3A%20false%0A%20%20%23%23%20Set%20to%20true%20to%20enable%20remote%20configuration.%0A%20%20enabled%3A%20true

But it looks like it worked well according to [the PoC result](https://www.notion.so/Datadog-as-IDS-IPS-and-FIM-109f205717b34e798bf4a69aab58997b?pvs=4#4918fbf279ec4ddf998a3d0c3527edeb) 🤔 
Please don't mind if it worked well, without  `remote_configuration:` parameter.

Reply:
You can see the complete config sample in the setting page of "Cloud Security Management".
In this page, click "Setup" and "Enable" link, then you can see the sample config.

https://app.datadoghq.com/security/configuration/csm/setup?active-step=hosts-table&missing_agent_capabilities=cws%7Ccspm%7Cvm_containers%7Cvm_hosts&operating_system=unix

Sorry for confusion, but It was complicated to show this page. So I pasted the document page. But it lacks `remote_configuration:` key.

Reply:
I see, that makes sense. Thank you!